bigquery/src/main/java/org/apache/zeppelin/bigquery/BigQueryInterpreter.java

```diff
@@ -17,43 +17,20 @@
 package org.apache.zeppelin.bigquery;
 
 
-import static org.apache.commons.lang.StringUtils.containsIgnoreCase;
-
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.GenericJson;
 import com.google.api.client.json.JsonFactory;
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.json.jackson2.JacksonFactory;
-
 import com.google.api.services.bigquery.Bigquery;
-import com.google.api.services.bigquery.BigqueryScopes;
-import com.google.api.client.json.GenericJson;
-import com.google.api.services.bigquery.Bigquery.Datasets;
-import com.google.api.services.bigquery.BigqueryRequest;
-import com.google.api.services.bigquery.model.DatasetList;
-import com.google.api.services.bigquery.model.Job;
-import com.google.api.services.bigquery.model.TableCell;
-import com.google.api.services.bigquery.model.TableFieldSchema;
-import com.google.api.services.bigquery.model.TableRow;
-import com.google.api.services.bigquery.model.TableSchema;
 import com.google.api.services.bigquery.Bigquery.Jobs.GetQueryResults;
-import com.google.api.services.bigquery.model.GetQueryResultsResponse;
-import com.google.api.services.bigquery.model.QueryRequest;
-import com.google.api.services.bigquery.model.QueryResponse;
-import com.google.api.services.bigquery.model.JobCancelResponse;
-import com.google.gson.Gson;
-
-import java.io.IOException;
-import java.util.Collection;
-import java.sql.ResultSet;
-import java.sql.ResultSetMetaData;
-import java.sql.SQLException;
-import java.util.Properties;
-import java.util.Set;
-
+import com.google.api.services.bigquery.BigqueryRequest;
+import com.google.api.services.bigquery.BigqueryScopes;
+import com.google.api.services.bigquery.model.*;
```

Comment:
I think this is trivial but Zeppelin don't recommend use * for imports. Please change your intellJ setting.